js/google_ima.js

```diff
@@ -147,6 +147,7 @@ require("../html5-common/js/utils/utils.js");
         this.adPlaybackStarted = false;
         this.vcPlayRequested = false;
         this.savedVolume = -1;
+        this.useGoogleAdUI = false;
```

Comment:
Because it's unproven and only 1 customer is requesting this. It'll be off by default.